backend/app/controllers/MenteesController.php

```diff
@@ -38,6 +42,101 @@ public function singleMentee($id) {
         ]);
     }
 
+    public function login()
+    {
+
+        $credentials = request()->get(['email', 'password']);
+
+        if(!$credentials['email'] || !$credentials['password']) {
+            response()->exit([
+                'error' => '404',
+                'message' => "One of required fields isn't filled.",
+            ]);
+        }
+        $user = User::where('email', $credentials['email'])->first();
+
+        $user = auth()->login($credentials);
+
+        if (!$user) {
+            response()->exit(auth()->errors());
+        }
+
+        response()->json($user);
+    }
+
+    public function menteeInfo()
+    {
+        if (auth()->user()) {
+            $id = auth()->id();
+
+            $menteeInfo = db()
+            ->select("users
+            LEFT JOIN users_info
+            ON users.id = users_info.users_id
+            WHERE id = ?")
+            ->hidden("password")
+            ->bind($id)
+            ->fetchAssoc();
+
+            response()->json([
+                "menteeinfo" => $menteeInfo,
+            ]);
+          } else {
+            response()->json([
+                "error" => 404,
+                "message" => "User isn't logged in.",
+            ]);
+          }
+    }
+
+    public function weightUpdate()
+    {
+        if (auth()->user()) {
+            $id = auth()->id();
+
+            $new_weight = request()->get('new_weight');
+
+            if(!$new_weight) {
+                response()->exit([
+                    "error" => 404,
+                    "message" => "Empty fields",
+                ]);
+            }
+
+            $weight_update = db()
+            ->update("users_info")
+            ->params(["current_weight" => $new_weight])
+            ->where("users_id", $id)
+            ->execute();
+
+            $weight_update_log = db()
+            ->insert("users_weights")
+            ->params([
+              "users_id" => $id,
+              "new_weight" => $new_weight
+            ])
+            ->execute();
+
+            if($weight_update && $weight_update_log) {
+                response()->json([
+                    "error" => 200,
+                    "newWeight" => $new_weight,
+                    "message" => "Weight update successfully.",
+                ]);
```

Comment:
We don't need newWeight in response as it's the same as in the request. This should be just a 204.

```php
response()->status(400);
```